docs/Wiki/ProductAndTestProjects.md

```diff
@@ -1,29 +1,62 @@
-﻿This page refers to the SonarScanner for .NET, also known as SonarScanner for MSBuild (S4MSB). You can read more about how to install and use it in the ["SonarScanner for .NET documentation"](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html).
+﻿This page refers to the SonarScanner for .NET (S4NET), also known as SonarScanner for MSBuild (S4MSB). You can read more about how to install and use it in the ["SonarScanner for .NET documentation"](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html).
+
+### Analyzer references
+
+* the `SonarAnalyzer.CSharp`/`SonarAnalyzer.VB` analyzers will be added to the build even if those analyzers are not referenced in the MSBuild project (S4NET will add references to those analyzers on the fly during the build). The set of rules that is executed is determined by the project type and Quality Profile for the Sonar project in SonarQube/SonarCloud.
+* any third-party analyzers referenced by the MSBuild project will be executed as part of the build (e.g. via NuGet package references). Issues from those analyzers will be uploaded to SonarQube/SonarCloud as [External Issues](https://sonarcloud.io/documentation/analysis/external-issues/). You can configure the third-party analyzer rules as normal using a `Ruleset`. During the build, S4NET will merge your custom ruleset with a ruleset generated from the Quality Profile. In the event of a conflict, the settings in the generated ruleset take precedence.
```

Comment:
I think we should document the fact that `.editorconfig` can be used, as well, for 3rd party analyzers. Not sure on its behavior with rulesets , though.

Reply:
As it's a Wiki, update is not bound to a release. We can improve it anytime. In this case, after Čaba finishes his investigation on the topic.